.github/workflows/publish-npm.yaml

```diff
@@ -0,0 +1,57 @@
+permissions: write-all
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - "*"
+
+jobs:
+  publish-to-npm:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
This shouldn't be needed, right?

Reply:
The checkout?  Not if we add it to the existing publish